platform/security/handler/security-handler-oidc/src/main/java/org/codice/ddf/security/handler/oidc/OidcHandlerConfigurationImpl.java

```diff
@@ -79,6 +80,11 @@ public void setProperties(Map<String, Object> properties) {
     responseMode = (String) properties.getOrDefault(RESPONSE_MODE_KEY, responseMode);
     logoutUri = (String) properties.getOrDefault(LOGOUT_URI_KEY, logoutUri);
 
+    // TODO - Remove if fragment response_mode is supported
+    if (IMPLICIT_FLOWS.contains(responseType)) {
```

Comment:
If a response_mode is unspecified when using implicit flow, Keycloak defaults to fragment which we don't support

Reply:
Can we also check that it is keycloak then?

Reply:
I wonder if the Authentication Server metadata returns the default (or supported) response_mode. If not, and especially since we only support form_post, I think defaulting to form_post is fine here.

Reply:
Yes let me just make sure it's not a default specified by the spec

Reply:
There is this part in the spec
```
For purposes of this specification, the default Response Mode for the OAuth 2.0 code Response Type is the query encoding. For purposes of this specification, the default Response Mode for the OAuth 2.0 token Response Type is the fragment encoding.
```

Reply:
Interesting that there are some other MUST rules buried in there. We'll want to make sure we have a good look at that when adjusting the config.